src/App/Hangouts/View.elm

```diff
@@ -1,44 +1,61 @@
 module App.Hangouts.View exposing (view)
 
 import Html exposing (..)
+import Paginate exposing (..)
 import Html.Attributes exposing (..)
-import App.Hangouts.Models exposing (Hangout, Group)
+import Html.Events exposing (onClick)
+import App.Hangouts.Models exposing (HangoutModel, Hangout, Group)
+import App.Hangouts.Messages exposing (Msg(..))
 import App.Auth.Models exposing (User)
 
 
-view : List Hangout -> Html msg
-view hangouts =
-    div [ class "table" ]
-        [ table []
-            (tr [] [ th [ colspan 4 ] [ text "HANGOUTS" ] ] :: hangoutsView hangouts)
-        , div [ class "hangout--button" ]
-            [ button []
-                [ img [ src "https://www.dropbox.com/s/okgmtdpih1xxau3/Shuffle.png?raw=1" ] []
-                , b [] [ text "SHUFFLE" ]
-                ]
-            ]
-        ]
+view : HangoutModel -> Html Msg
+view hangoutModel =
+    let
+        buttonClass =
+            "button is-primary is-outlined"
 
+        isShuffling =
+            if hangoutModel.loading == True then
```

Comment:
Cleaner if we change it to `if hangoutModel.loading then ...` since `loading` is a `Bool` already.